### ISSUE TITLE
Missing super component registration flow

### ISSUE THREAD

**Issue by a995049470** [ncannasse/domkit#30]
when i use domkit , sometimes  prompt " Missing super component registration flow". it is not a ture error. after i restart vscode , will report the true error.
![image](https://user-images.githubusercontent.com/37652153/111068441-b7348280-8503-11eb-810c-ff70509183cf.png)
![image](https://user-images.githubusercontent.com/37652153/111068454-be5b9080-8503-11eb-91bc-e6826a15dca0.png)

**Comment by ncannasse** [comment]
Yes it's a known issue related to the compiler cache and components meta classes. Haven't yet been able to fully reproduce and investigate it

**Comment by a995049470** [comment]
is have a simple way to avoid it ?
